mod/developers/views/default/js/elgg/dev.js

```diff
@@ -2,10 +2,11 @@
  * Elgg developers tool JavaScript
  */
 define(function(require) {
-	var $ = require('jquery'); require('jquery.jstree');
+	var $ = require('jquery');
 
-	var setupInspectTree = function() {
-		
+	require('jquery.jstree');
+
+	function setupInspectTree() {
```

Comment:
This file can be or extracted to a separate commit/PR. It's cleanup that isn't relevant to this change.